direct/core/src/main/java/cz/o2/proxima/direct/view/LocalCachedPartitionedView.java

```diff
@@ -167,10 +172,15 @@ public void onCompleted() {
     CommitLogObserver observer =
         new CommitLogObserver() {
 
+          long lastCleanup = 0;
```

Comment:
nit: Long.MIN_VALUE

Reply:
This is on purpose 0, because the MIN_VALUE might cause issues when subtracting.